sys/uart_stdio/uart_stdio.c

```diff
@@ -109,7 +122,13 @@ void uart_stdio_init(void)
 
 int uart_stdio_read(char* buffer, int count)
 {
+#ifdef UART_STDIO_RX
     return isrpipe_read(&uart_stdio_isrpipe, buffer, count);
+#else
+    (void)buffer;
+    (void)count;
+    return -ENOTSUP;
```

Comment:
Just wondering if this has any side effects on existing applications?

Reply:
Difficult question... Maybe returning count is the better choice...